vendor/github.com/tsg/gopacket/pcap/pcap.go

```diff
@@ -170,7 +170,7 @@ type InterfaceAddress struct {
 // BPF is a compiled filter program, useful for offline packet matching.
 type BPF struct {
 	orig string
-	bpf  _Ctype_struct_bpf_program // takes a finalizer, not overriden by outsiders
+	bpf  C.struct_bpf_program // takes a finalizer, not overriden by outsiders
```

Comment:
Not really familiar with this change. @tsg could you have a look?

Reply:
He merged the PR in his own repo. So he already accepted it. :)